clientaway.cpp

```diff
@@ -84,21 +84,45 @@ class CClientAwayMod : public CModule {
 		const vector<CClient*> vClients = m_pUser->GetAllClients();
 
 		CString sHostname = sLine.Token(1);
+
+		// Valid value of true or false for setting away/unaway
+		bool sToggleFlag = true;
+		CString sToggleValue = "away";
+        VCString sReturn;
```

Comment:
Looks like the indentation here was changed, all of ZNC source code uses tabs and not spaces and this module is consistent with that.

Would you be able to amend this to match (switch to tabs)?